interface-elements-for-desktop/articles/rich-text-editor/track-changes/enable-track-changes.md

```diff
@@ -0,0 +1,59 @@
+# Enable Track Changes
+
+The Rich Text Editor allows you to use Track Changes to make revisions in a document.
```

Comment:
to use Track Changes to make revisions? The 'Track Changes' feature allows you to keep track of the changes, to see which changes were made

Reply:
changed